test/functional/cypress/specs/events/filter-spec.js

```diff
@@ -558,6 +558,46 @@ describe('events Collections Filter', () => {
         cy.get(latestStartElement).clear()
       })
     })
+
+    context('Aventri ID', () => {
+      const element = '[data-test="aventri-id-filter"]'
+      const aventriId = '200300400'
+      const invalidAventriId = 'Testing'
```

Comment:
Maybe add a space and a symbol on the end just to double check it is not allowing those too!
```suggestion
      const invalidAventriId = 'Testing %'
```